pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -480,6 +480,17 @@ def test_to_datetime_unparseable_ignore(self):
         s = "Month 1, 1999"
         assert pd.to_datetime(s, errors="ignore") == s
 
+    def test_to_datetime_unparseable_raise(self):
+        # GH#10720
+        invalid_data = "Month 1, 1999"
+        expected_args = (
+            f"Unknown string format: {invalid_data}. "
+            f"You can coerce to NaT by passing errors='coerce'"
```

Comment:
```suggestion
            "You can coerce to NaT by passing errors='coerce'"
```

Don't need the f prefix if not parametrizing anything